test/kubemark/iks/util.sh

```diff
@@ -164,23 +168,25 @@ function generate-values {
 	echo "Generating values"
 	master-config
 	KUBECTL=kubectl
+	export KUBECTL
```

Comment:
As you said above, you'll want to add a comment about this and `TEST_CLUSTER_API_CONTENT_TYPE` to `complete-login`